scripts/slurm/compile.bessemer.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#SBATCH --time=00:30:00
+#SBATCH --ntasks=4
+#SBATCH --job-name=compile.bessemer.sh
+
+# Load modules for compilation
+module use /usr/local/modulefiles/staging/eb/all/
+module load CUDA/11.0.2-GCC-9.3.0
+
+# Load cmake with matching glibc version. Alternatively use a cmake installed via pip into a local venv.
+module load CMake/3.16.4-GCCcore-9.3.0
```

Comment:
This is insufficient to build recent flame. @MILeach used a local pip install instead, so this will need updating for that.

Reply:
@MILeach Can you check this please and commit a change if required.